### PR TITLE
Bbowman/packaging merge fix

### PR DESCRIPTION
**Pull request by bbowman**
Pull in latest develop and fix issues that came up from the merge.

### CODE REVIEW

**Comment by bbowman** [comment]
The last commit is the interesting one (fixing merge conflicts). Dunno the best way to review it.

File: include/Platform/Universal Windows/UWP/RTHelpers.h [hunk 1]
```diff
@@ -51,8 +51,8 @@
 #include <UWP/interopBase.h>
 
 #if __has_feature(objc_arc)
-#define AUTORELEASE(x)
-#define RETAIN(x)
+#define AUTORELEASE(x) x
```
Comment (memontic-ms) [inline include/Platform/Universal Windows/UWP/RTHelpers.h]: There's an extra x floating around on the two lines. Don't see that in develop or packaging.

File: build/Tests/UnitTests/Accelerate/Accelerate.UnitTests.vcxproj [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (memontic-ms) [inline build/Tests/UnitTests/Accelerate/Accelerate.UnitTests.vcxproj]: Might want to roll these all back to 14.0 for now?

File: include/Platform/Universal Windows/UWP/RTHelpers.h (hunk 1 above)
Comment (bbowman) [inline include/Platform/Universal Windows/UWP/RTHelpers.h]: They are the "return" values here. some of these guys are supposed to return themselves so you can [[foo retain] bar ] which was creating errors somewhere. I was surprised it worked before.

File: build/Tests/UnitTests/Accelerate/Accelerate.UnitTests.vcxproj (hunk 2 above)
Comment (bbowman) [inline build/Tests/UnitTests/Accelerate/Accelerate.UnitTests.vcxproj]: done

File: include/Platform/Universal Windows/UWP/RTHelpers.h (hunk 1 above)
Comment (memontic-ms) [inline include/Platform/Universal Windows/UWP/RTHelpers.h]: Sounds good. I was concerned it was a merge error.

**Comment by memontic-ms** [comment]
:shipit:

**Pull request merged**